presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -93,4 +95,30 @@ public GlueHiveMetastoreConfig setIamRole(String iamRole)
         this.iamRole = Optional.ofNullable(iamRole);
         return this;
     }
+
+    public Optional<String> getAwsAccessKey()
+    {
+        return awsAccessKey;
+    }
+
+    @Config("hive.metastore.glue.aws-access-key")
+    @ConfigDescription("Hive Glue metastore AWS access key")
+    public GlueHiveMetastoreConfig setAwsAccessKey(String awsAccessKey)
+    {
+        this.awsAccessKey = Optional.ofNullable(awsAccessKey);
+        return this;
+    }
+
+    public Optional<String> getAwsSecretKey()
+    {
+        return awsSecretKey;
+    }
+
+    @Config("hive.metastore.glue.aws-secret-key")
+    @ConfigDescription("Hive Glue metastore AWS secret key")
```

Comment:
Please add `@ConfigSecuritySensitive`. The value should not be printed to server.log during server startup.